### DISCUSSION TITLE
How do you build the sample application with cmake on linux

### DISCUSSION THREAD

**Discussion by vijayee** [microsoft/msquic#3352] [Q&A] [answered]
I'm trying to run the sample code on Linux compiled with clang. Is there any build instructions for this?

**Comment by nibanks** [comment, marked as answer]
To build the sample (or any tools), you'll need to set the `QUIC_BUILD_TOOLS` cmake flag to `on`.

**Reply by vijayee** [reply]
Thanks, that worked
